test/unittest/base/test_jsonization.py

```diff
@@ -0,0 +1,70 @@
+# import superduperdb as s
+
+import typing as t
+
+import pydantic as pd
```

Comment:
We should avoid this alias - `pd` is reserved for `pandas` as far as I am concerned.